### PR TITLE
Png update

### PR DESCRIPTION
**Pull request by randyh62**
Convert JPG files to PNG format
Remove JP files from docs/data/doxygenInputs and docs/data/doxygenOutputs folder
update header files to point to PNG image files

### CODE REVIEW

**Comment by kiritigowda** [comment]
@r-abishek can you review this PR and check if the standalone doc generation works?

**Comment by r-abishek** [comment]
@kiritigowda Verified Local machine Doxygen build still shows all images as below after the jpg->png conversion
image

**Review by r-abishek — APPROVED** [review]

File: docs/doxygen/Doxyfile [hunk 1]
```diff
@@ -1117,8 +1117,8 @@ EXAMPLE_RECURSIVE      = NO
 # that contain images that are to be included in the documentation (see the
 # \image command).
 
-IMAGE_PATH             = ../../data/doxygenInputs \
-                         ../../data/doxygenOutputs
+IMAGE_PATH             = ../data/doxygenInputs \
```
Comment (r-abishek) [inline docs/doxygen/Doxyfile:1120]: Interesting that the path change didn't affect anything on local build. Images show before and now.

Comment (randyh62) [reply docs/doxygen/Doxyfile:1120]: It was interesting that it built with the ../../ path when you consider the Doxyfile is in docs/doxygen and the images are in docs/data. It did not build correctly on my system, which is why I changed it.

**Pull request merged**